node/src/main/scala/com/wavesplatform/state/diffs/TransactionDiffer.scala

```
@@ -301,7 +301,7 @@ object TransactionDiffer {
         portfolios = portfolios,
         scriptResults = scriptResult.fold(Map.empty[ByteStr, InvokeScriptResult])(sr => Map(tx.id() -> sr)),
         scriptsComplexity = spentComplexity
-      ).unsafeCombine(ethereumMetaDiff)
+      ).combineF(ethereumMetaDiff).getOrElse(Diff.empty)
```

Should we silently ignore the error?